src/main/java/edu/cornell/kfs/concur/businessobjects/ValidationResult.java

```diff
@@ -0,0 +1,52 @@
+package edu.cornell.kfs.concur.businessobjects;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ValidationResult {
+
+    protected boolean valid;
+    protected List<String> messages;
+
+    public ValidationResult(boolean valid, List<String> messages) {
+        this.valid = valid;
+        this.messages = messages;
+    }
+
+    public boolean isValid() {
+        return valid;
+    }
+
+    public void setValid(boolean valid) {
+        this.valid = valid;
+    }
+
+    public List<String> getMessages() {
+        return messages;
+    }
+
+    public void setMessages(List<String> messages) {
+        this.messages = messages;
+    }
+
+    public void addMessage(String message) {
+        if (messages == null) {
+            messages = new ArrayList<String>();
+        }
+        messages.add(message);
+    }
+
+    @Override
+    public String toString() {
+        StringBuffer result = new StringBuffer();
```

Comment:
I think I better understand your concern now. Yes, my plan was to use the toString method to get all the error messages in a formatter String that is sent back to Concur. I think it would be best to have a new method implemented for this and not use toString. I will update the code.